src/libs/CollectionUtils.ts

```diff
@@ -0,0 +1,23 @@
+import {OnyxCollectionKey} from '../ONYXKEYS';
+
+/**
+ * Return the highest item in a numbered collection
+ *
+ * e.g. {1: '1', 2: '2', 3: '3'} -> '3'
+ *
+ * Use this only for collections that are numbered in other cases it will return the last item in the object not the highest
+ */
+function lastItem<T>(object: Record<string, T> = {}): T | undefined {
+    const lastKey = Object.keys(object).pop() ?? 0;
```

Comment:
Is this tested? We have to make sure it is an equivalent of how it was implemented with underscore @kubabutkiewicz

Reply:
@blazejkustra Yes, I tested that and results are the same.
Also quick [JSFiddle](https://jsfiddle.net/zsa5vwnt/8/) test to compare output